README.adoc

```diff
@@ -307,17 +334,36 @@ output:
  T E S T S
 -------------------------------------------------------
 Running it.io.openliberty.guides.consumingrest.ConsumingRestTest
-Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.957 sec - in it.io.openliberty.guides.consumingrest.ConsumingRestTest
+Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 1.59 sec - in it.io.openliberty.guides.consumingrest.ConsumingRestTest
 
 Results :
 
-Tests run: 3, Failures: 0, Errors: 0, Skipped: 0
+Tests run: 4, Failures: 0, Errors: 0, Skipped: 0
+
 ----
 
+== Conclusions
```

Comment:
We generally don't include any kind of conclusion section in our guides. If this information is necessary to include in the guide, could you place it earlier in the file in another section? Perhaps you could create a new section with its own title and put the new section around the sections on "POJOs" and "Using JSON-B" at the beginning of the guide.